generators/entity-client/files.js

```diff
@@ -455,12 +455,14 @@ function writeFiles() {
             let clientMainSrcDir;
             let templatesDir;
             let microserviceName = this.microserviceName;
+            let pageTitle;
 
             if (this.clientFramework === ANGULAR) {
                 files = angularFiles;
                 clientMainSrcDir = ANGULAR_DIR;
                 templatesDir = CLIENT_NG2_TEMPLATES_DIR;
                 microserviceName = this.microserviceName;
+                pageTitle = this.enableTranslation ? `${this.i18nKeyPrefix}.home.title` : this.entityClassPlural;
```

Comment:
Not needed

Reply:
You are right, changes here not needed, because in `generator-base.js` in  `addEntityToModule` entity context is available. But question is: why we at all need all those parameters to `addEntityToModule`. Maybe it would be easier to have `addEntityToModule` with zero parameters and read all other information from entity context also there?
Why to act differently with `pageTitle` parameter?
Anyway, for now reverting changes here and using
```
this.enableTranslation ? `${this.i18nKeyPrefix}.home.title` : this.entityClassPlural
```
in internal Angular method in PR coming shortly.

Reply:
> why we at all need all those parameters to addEntityToModule

This is actually a code smell, it should be an object instead.

> Maybe it would be easier to have addEntityToModule with zero parameters and read all other information from entity context also there?

context could be an optional parameter (defaults to this.context), but not sure context contains every corresponding parameters.
It's important to don't make `this.context` mandatory so modules/blueprints can add modules without need to create a this.context.

Reply:
In #13198 gave default values to all `addEntityToModule` parameters so that here can use zero parameters and modules/blueprints can give all parameters - so use without creating a context.